modules/apps/company/company-test/src/testIntegration/java/com/liferay/company/service/test/CompanyLocalServiceDBPartitionTest.java

```diff
@@ -377,10 +406,8 @@ public void testAddDBPartitionCompanyWhenDBPartitionUtilFails()
 	public void testCopyDBPartitionCompany() throws Exception {
 		int rulesCount = _getRulesCount(defaultPartitionName);
 
-		Company company = CompanyTestUtil.addCompany();
```

Comment:
We should not mix different changes. Let's keep the creation of the company and remove in a separate ticket for the whole class.